app/controllers/ems_configuration_controller.rb

```diff
@@ -0,0 +1,103 @@
+class EmsConfigurationController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::BreadcrumbsMixin
+  include Mixins::GenericFormMixin
+  include Mixins::ManagerControllerMixin
+  include Mixins::FindRecord
+
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def self.model
+    ManageIQ::Providers::ConfigurationManager
+  end
+
+  def self.table_name
+    @table_name ||= "ems_configuration"
+  end
+
+  def self.display_methods
+    %w[configuration_profiles configured_systems]
+  end
+
+  def button
+    @edit = session[:edit] # Restore @edit for adv search box
+    params[:display] = @display if %w[configured_systems].include?(@display) # Were we displaying nested list
```

Comment:
same here (`if display_methods.include?(@display)`), but what happens for nested lists of configuration_profiles?